ml-agents-envs/mlagents_envs/gym_wrapper.py

```diff
@@ -0,0 +1,163 @@
+from mlagents_envs.base_env import (
+    BaseEnv,
+    DecisionSteps,
+    TerminalSteps,
+    BehaviorSpec,
+    BehaviorName,
+    AgentId,
+    ActionType,
+)
+from mlagents_envs.exception import UnityActionException, UnityObservationException
+
+from typing import List, Tuple
+
+import numpy as np
+
+import gym
+
+
+class GymWrapper(BaseEnv):
```

Comment:
I think you need a different name besides "wrapper", since that's what we already call gym-unity: https://github.com/Unity-Technologies/ml-agents/tree/master/gym-unity#unity-ml-agents-gym-wrapper
Maybe GymAdapter?

Reply:
Or be more explicit : GymToBaseEnvWrapper ?

Reply:
and rename the other BaseEnvToGymWrapper?

Reply:
Tangential: we need a name for the LL-API that isn't `BaseEnv` or `LL-API`. 😅

Reply:
UnityEnvToGymWrapper?

Reply:
#3812